src/raids/uldir/Ghuun.js

```diff
@@ -18,5 +21,35 @@ export default {
       ],
       magical: [],
     },
+    phases: [
+      {
+        id: '1',
```

Comment:
can we make this an int?

Reply:
I did originally, but when I came to implementing the micro phases, though having them as a string would allow for more customisation. This also works better for things like the chambers on Mother if we wanted to do those with id's of 'C1', 'C2' and 'C3'

At least that was my idea...

Reply:
Maybe then it should be an int. What is the id used for?

Reply:
I only use it in one place at the moment, and that's for matching the end event of add waves to the start. It could probably be used in the future for filtering the data down too if you only want to parse events for a particular phase

Reply:
Then it should probably be the key of an object instead